lib/takeaway_class.rb

```diff
@@ -0,0 +1,38 @@
+class Takeaway
+
+	attr_reader :menu, :order, :actualcost
+
+	def initialize
+		@menu = {"Cheeseburger" => 1,
+			"Pizza" => 2,
+			"Fries" => 3
+		}
+		@actualcost = 0
+	end
```

Comment:
Ruby convention is 2 space indentation. Clean and readable code is really important moving forward. Try adjusting the settings in your text editor .